frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -3584,9 +3584,30 @@ static bool resolveFnCallSpecial(Context* context,
                                  const AstNode* astForErr,
                                  const CallInfo& ci,
                                  QualifiedType& exprTypeOut) {
-  // TODO: .borrow()
   // TODO: chpl__coerceCopy
 
+  // Consider the resolution of .borrow() and release here a performance optimization, other
+  // methods on _owned should be resolved the "normal" way, through the module code
+  if (ci.isMethodCall() &&
+      (ci.name() == UniqueString::get(context, "borrow") ||
+       ci.name() == UniqueString::get(context, "release"))) {
```

Comment:
I'm not sure if `release` should be resolved here the same as `borrow`. If I remove this extra condition then calls to `owned.release(myObj)` start failing with `The formal 'this' expects the type '_owned', but the actual was the type 'owned'.`

Reply:
yes, it does. If I add some logic to `tryConvertClassTypeIntoManagerRecordIfNeeded` to handle the case where `mightBeClassType` can also be `AnyOwnedType` it will get past this, but it will fail due to ambiguity.